workspaces/src/types/mod.rs

```diff
@@ -44,6 +44,9 @@ pub type BlockHeight = u64;
 /// Shard index, from 0 to NUM_SHARDS - 1.
 pub type ShardId = u64;
 
+// Constant taken from nearcore crate to avoid dependency
+pub const DEFAULT_DEPOSIT: NearToken = NearToken::from_near(100);
```

Comment:
There is no sense in such a public constant. There is no such a thing like "default deposit" in NEAR protocol.

Reply:
I moved this over to types crate: https://github.com/near/near-workspaces-rs/blob/51e0ead85556c57f3bd41398cd75025ad51c33b3/workspaces/src/network/sandbox.rs#L22

Reply:
> There is no sense in such a public constant. There is no such a thing like "default deposit" in NEAR protocol.

Updated the docs and renamed the constant to reflect that.